larq/conftest.py

```diff
@@ -46,3 +52,12 @@ def keras_should_run_eagerly(request):
     else:
         with context.eager_mode():
             yield request.param == "tf_keras_eager"
+
+
+@pytest.fixture(params=[False, True])
```

Comment:
It might be nicer to make these params strings describing what the different settings are instead of booleans, mirroring previous fixtures:
- `eager_and_graph_mode` (`["eager", "graph"]`) and
- `keras_should_run_eagerly ` (`["graph", "tf_eager", "tf_keras_eager"]`)

Reply:
I opted for booleans so it is easy to check whether we are running the unittest distributed (see [here](https://github.com/larq/larq/pull/312/files#diff-2c6d294a8f4ae0b3b2ee3df2966dca03R31))

Reply:
OK makes sense.